test/Altinn.Platform.Events.Tests/TestingServices/EventsServiceTest.cs

```diff
@@ -38,6 +41,10 @@ public EventsServiceTest()
             _authorizationMock = new();
             _claimsPrincipalProviderMock = new();
             _loggerMock = new();
+            _serializerOptions = new()
+            {
+                PropertyNamingPolicy = JsonNamingPolicy.CamelCase
```

Comment:
Found this interesting point in the spec regarding casing:
https://github.com/cloudevents/spec/blob/v1.0.2/cloudevents/spec.md#attribute-naming-convention

Don't have to do it now, but it would be great to look more into what future pain we can avoid by using the cloudevents.io SDK rather than handling serialization ourselves.

Reply:
A small note: The configuration used here will be overridden by the attributes decorating each property in the model(s).

Reply:
using a new jsonSerializer instead. Not possible to set to null in the validation within a mock implementation